### DISCUSSION TITLE
add an option to apply quick fix on save

### DISCUSSION THREAD

**Discussion by scarf005** [rome/tools#3856] [Suggestions]
![_01](https://user-images.githubusercontent.com/54838975/203994195-6e917674-6f77-4349-928f-0e0ed186e72f.png)
because, it would be nicer than manually typing `rome check --apply-suggested .`

**Comment by leops** [comment]
This is something you should be able to do once we release v11 by adding `"editor.codeActionsOnSave": { "quickfix.rome": true }` to you editor configuration

**Reply by jpike88** [reply]
@leops is there an issue tracking this?
